architecture/tube.go

```diff
@@ -213,3 +214,33 @@ func (tube *Tube) Bury(command *Command) {
 		command.Err = errors.New(NOT_FOUND)
 	}
 }
+
+func (tube *Tube) Kick(command *Command) {
+	amount := 0
+	bound, err := strconv.Atoi(command.Params["bound"])
+	if err != nil {
+		// handle non-integer number of jobs to kick
```

Comment:
Log an error here, set the following to `command.Err` 
`command.Err = errors.New(NOT_FOUND)`
and return from function (`NOT_FOUND` is the only error message allowed for kick https://github.com/vimukthi-git/beanstalkg/blob/master/protocol.md), 
It doesn't make sense to process the command afterwards right?